redbot/cogs/warnings/warnings.py

```diff
@@ -422,6 +424,29 @@ def __init__(self, bot: Red):
         if user_id == ctx.author.id:
             return await ctx.send(_("You cannot remove warnings from yourself."))
 
+        custom_allowed = await self.config.guild(ctx.guild).allow_custom_reasons()
+        guild_settings = self.config.guild(ctx.guild)
+        reason_type = None
+        async with guild_settings.reasons() as registered_reasons:
+            if reason.lower() not in registered_reasons:
+                msg = _("That is not a registered reason!")
```

Comment:
```suggestion
                msg = _("That is not a registered reason.")
```
Please no shouting is too early in the morning.